Makefile

```diff
@@ -0,0 +1,13 @@
+help:
+	@echo "start - starts all services"
+	@echo "stop  - stops all services and removes anonymous volumes"
+	@echo "shell - runs an instance of the 'app' service and presents a bash prompt"
+
+start:
+	docker-compose up
```

Comment:
consider an explicit `--build` here

Reply:
I had but couldn't find a reason for it at this moment.

Reply:
The main difference is `make start` won't see changes to the files in the image (requiring a separate `docker-compose build && make start`) but the only ones that this applies to at the moment are `requirements/*` files.